examples/slash/slash.go

```diff
@@ -11,24 +11,36 @@ import (
 
 func main() {
 	var (
-		verificationToken string
+		signingSecret string
 	)
 
-	flag.StringVar(&verificationToken, "token", "YOUR_VERIFICATION_TOKEN_HERE", "Your Slash Verification Token")
+	flag.StringVar(&signingSecret, "secret", "YOUR_SIGNING_SECRET_HERE", "Your Slack app's signing secret")
 	flag.Parse()
 
 	http.HandleFunc("/slash", func(w http.ResponseWriter, r *http.Request) {
-		s, err := slack.SlashCommandParse(r)
+
+		verifier, err := slack.NewSecretsVerifier(r.Header, signingSecret)
 		if err != nil {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
 
-		if !s.ValidateToken(verificationToken) {
+		_, err = verifier.WriteBody(r)
```

Comment:
new method should be unnecessary.

```suggestion
		if _, err = io.Copy(verifier, r); err != nil {
```